### DISCUSSION TITLE
Redux and Redux Saga

### DISCUSSION THREAD

**Discussion by iqbal-afrianto** [mmazzarolo/react-native-universal-monorepo#19] [Q&A] [answered]
Hi i've been learning this monorepo setup and its really cool!

I have some questions, if i want to integrate the app with redux and redux saga, is it enough to just configure it on the ``` app ``` workspace? or should i also configure it on the other workspace too?

Thanks

**Comment by mmazzarolo** [comment, marked as answer]
Hey! Yes, all the JavaScript-only (or TypeScript-only) dependencies (meaning they don't need to be linked to the native code) can be added just to the `app` workspace (or to any additional JavaScript workspace you'll add in the future) 👍
